lib/improver/utilities/cube_manipulation.py

```diff
@@ -249,6 +261,13 @@ def process(self, cubes_in):
             ValueError:
                 If master coordinate is not present on all "cubes_in"
         """
+        # if input is a single cube or list of length 1, return unchanged
+        if isinstance(cubes_in, iris.cube.Cube):
+            return cubes_in
+
+        if len(cubes_in) == 1:
+            return cubes_in[0]
+
         # create copies of input cubes so as not to modify in place
         if isinstance(cubes_in, iris.cube.Cube):
```

Comment:
It won't get into this if statement now because cubes_in will have already been returned?

Reply:
True :)  Will tidy this up.